modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -17,11 +17,31 @@ You can also use the Agent-based Installer to generate or accept Zero Touch Prov
 * You read the documentation on xref:../../installing/installing-preparing.html#installing-preparing-selecting-cluster-type[Selecting a cluster installation method and preparing it for users].
 * You have read the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[Preparing to install with the Agent-based Installer] documentation.
 * You downloaded the xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Agent-Based Installer] and the command-line interface (CLI) from Red Hat’s Hybrid Cloud Console.
-* For a disconnected environment, you created a container image registry, such as Red Hat Quay. See xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-introduction_installing-mirroring-creating-registry[Mirror registry for Red Hat OpenShift introduction].
 * You have logged into the {product-title} with administrator privileges. 
 
 .Procedure
 
+. For a disconnected environment, you mirrored the {product-mirror-registry} to your local container image registry. See xref:../../installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the {product-title} image repository].
++
+[IMPORTANT]
+====
+Check that your `openshift-install` binary version relates to your local image container registry and not a shared registry such as {quay} as demonsrated in the following example:
+
+[source,terminal]
+----
+$ ./<path_to_openshift_installer_binary>/openshift-install version
+----
+
+Example output
+[source,terminal]
+----
+./<path_to_openshift_installer_binary>/openshift-install version 4.14.9
+built from commit <commit_id>
+virthost.ostest.test.metalkube.org:5000/localimages/local-release-image@sha256:<sha_256_id>
```

Comment:
Instead of using virthost.ostest.test.metalkube.org:5000, I would suggest an example that refers to their registry's domain name. Maybe you have a better way to word it:
```suggestion
<your-registry.domain:port>/localimages/local-release-image@sha256:<sha_256_id>
```

Reply:
Thanks. Updated, but maybe now the example is meaningless as it just ahs many user-repalced values.